_migration-assistant/migration-phases/planning-your-migration/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Planning your migration
+nav_order: 59
+parent: Migration phases
+has_children: true 
```

Comment:
Set `has_toc` to false so related articles don't show. Otherwise, the steps are listed twice.